tests/bluetooth/tester/src/btp_ias.c

```diff
@@ -0,0 +1,62 @@
+/* btp_ias.c - Bluetooth IAS Server Tester */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/bluetooth/services/ias.h>
+
+#include "btp/btp.h"
+#include "zephyr/sys/byteorder.h"
+
+#include <zephyr/logging/log.h>
+#define LOG_MODULE_NAME bttester_ias
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#define CONTROLLER_INDEX 0
+
+static enum bt_ias_alert_lvl g_ias_alert_lvl;
+
+/* Immediate Alert Service */
+static void alert_stop(void)
+{
+	struct ias_alert_action_ev ev;
+
+	g_ias_alert_lvl = BT_IAS_ALERT_LVL_NO_ALERT;
+
+	ev.alert_lvl = sys_cpu_to_le32(g_ias_alert_lvl);
+
+	tester_send(BTP_SERVICE_ID_IAS, IAS_EV_OUT_ALERT_ACTION,
+		    CONTROLLER_INDEX, (uint8_t *)&ev, sizeof(ev));
```

Comment:
I know that `ias_alert_action_ev` is packed, but the underlying `enum` isn't. 

What value does `sizeof(ev)` actually return, and is it the value that is expected? 

Generally using `sizeof` on `enum`s and expecting a specific size, isn't a good idea (even though it may have been my idea to begin with).

Perhaps it is better to leave `ias_alert_action_ev.alert_lvl` as an `uint8_t`?

Reply:
yeap, this should be uint8_t and BTP needs to specify values for it (see my comment in struct definition)

Reply:
done